projects/1.18/assets/applied-energistics-2/ae2/lang/zh_cn.json

```
@@ -1 +1,795 @@
+  "block.ae2.quartz_pillar_slab": "竖纹赛特斯石英台阶",
+  "block.ae2.quartz_pillar_stairs": "竖纹赛特斯石英楼梯",
+  "block.ae2.quartz_pillar_wall": "竖纹赛特斯石英墙",
+  "block.ae2.quartz_slab": "赛特斯石英台阶",
+  "block.ae2.quartz_stairs": "赛特斯石英楼梯",
+  "block.ae2.quartz_vibrant_glass": "聚能石英玻璃",
+  "block.ae2.quartz_wall": "赛特斯石英墙",
+  "block.ae2.security_station": "ME安全终端",
+  "block.ae2.sky_compass": "陨石罗盘",
+  "block.ae2.sky_stone_block": "陨石",
+  "block.ae2.sky_stone_brick": "陨石石砖",
+  "block.ae2.sky_stone_brick_slab": "陨石石砖台阶",
+  "block.ae2.sky_stone_brick_stairs": "陨石石砖楼梯",
+  "block.ae2.sky_stone_brick_wall": "陨石石砖墙",
+  "block.ae2.sky_stone_chest": "陨石箱子",
+  "block.ae2.sky_stone_slab": "陨石台阶",
+  "block.ae2.sky_stone_small_brick": "陨石砖块",
+  "block.ae2.sky_stone_small_brick_slab": "陨石砖块台阶",
+  "block.ae2.sky_stone_small_brick_stairs": "陨石砖块楼梯",
+  "block.ae2.sky_stone_small_brick_wall": "陨石砖块墙",
+  "block.ae2.sky_stone_stairs": "陨石楼梯",
+  "block.ae2.sky_stone_tank": "陨石储罐",
+  "block.ae2.sky_stone_wall": "陨石墙",
+  "block.ae2.smooth_sky_stone_block": "陨石块",
+  "block.ae2.smooth_sky_stone_chest": "陨石块箱子",
+  "block.ae2.smooth_sky_stone_slab": "陨石块台阶",
+  "block.ae2.smooth_sky_stone_stairs": "陨石块楼梯",
+  "block.ae2.smooth_sky_stone_wall": "陨石块墙",
+  "block.ae2.spatial_anchor": "空间锚",
+  "block.ae2.spatial_io_port": "空间IO端口",
+  "block.ae2.spatial_pylon": "空间塔",
+  "block.ae2.tiny_tnt": "迷你TNT",
+  "block.ae2.vibration_chamber": "谐振仓",
+  "block.ae2.wireless_access_point": "ME无线访问点",
+  "chat.ae2.AmmoDepleted": "弹药耗尽",
+  "chat.ae2.ChestCannotReadStorageCell": "ME箱子无法读取存储元件",
+  "chat.ae2.CommunicationError": "网络交互错误",
+  "chat.ae2.DeviceNotLinked": "设备未连接",
+  "chat.ae2.DeviceNotPowered": "设备供能不足",
+  "chat.ae2.InvalidMachine": "无效机器",
+  "chat.ae2.LoadedSettings": "成功载入设定",
+  "chat.ae2.MachineNotPowered": "机器未供能",
+  "chat.ae2.MaxUpgradesInstalled": "机器无法插入更多升级卡",
+  "chat.ae2.MaxUpgradesOfTypeInstalled": "机器无法再插入该类升级卡",
+  "chat.ae2.OnlyEmptyCellsCanBeDisassembled": "请先清空元件再拆解",
+  "chat.ae2.OutOfRange": "超出无线信号范围",
+  "chat.ae2.ResetSettings": "新的设备配置已被创建并保存到内存卡中",
+  "chat.ae2.SavedSettings": "已将当前设备的配置保存到内存卡中",
+  "chat.ae2.SettingCleared": "设置已清除",
+  "chat.ae2.StationCanNotBeLocated": "无法定位安全终端",
+  "chat.ae2.UnsupportedUpgrade": "机器不支持此类升级卡",
+  "chat.ae2.isNowLocked": "监控器已锁定",
+  "chat.ae2.isNowUnlocked": "监控器已解锁",
+  "commands.ae2.ChunkLoggerOff": "区块日志已关闭",
+  "commands.ae2.ChunkLoggerOn": "区块日志已打开",
+  "commands.ae2.permissions": "你没有足够的权限来执行该指令",
+  "commands.ae2.usage": "ae2提供的指令 - 输入/ae2 list查看指令列表，/ae2 help _____ 查看指令帮助.",
+  "entity.ae2.charged_quartz": "充能赛特斯石英水晶",
+  "entity.ae2.growing_crystal": "水晶种子",
+  "entity.ae2.singularity": "奇点",
+  "entity.ae2.tiny_tnt_primed": "被激活的迷你TNT",
+  "gui.ae2.And": "与",
+  "gui.ae2.AttachedTo": "连接到: %s",
+  "gui.ae2.Automatic": "自动分配",
+  "gui.ae2.Black": "黑色",
+  "gui.ae2.Blank": "空白",
+  "gui.ae2.Blue": "蓝色",
+  "gui.ae2.Brown": "棕色",
+  "gui.ae2.BytesUsed": "%s 字节已使用",
```

不知道删空格渲染会不会炸
并且搞忘%s里面是什么了，或许把“已使用”前置？
```suggestion
  "gui.ae2.BytesUsed": "%s字节已使用",
```